verif/sim/cva6.py

```diff
@@ -879,14 +879,14 @@ def load_config(args, cwd):
       args.isa  = "rv64gch_zba_zbb_zbs_zbc"
     elif base in ("cv64a6_imafdc_sv39", "cv64a6_imafdc_sv39_hpdcache", "cv64a6_imafdc_sv39_wb"):
       args.mabi = "lp64d"
-      args.isa  = "rv64gc_zba_zbb_zbs_zbc"
```

Comment:
cv64a6_imafdc_sv39_wb does not support zbkb, this has to be fixed

Reply:
we want to enable this extension for the basic 64-bit config as well, which is `cv64a6_imafdc_sv39` 

@munailwaqar lets only enable it for cv64a6_imafdc_sv39.